### DISCUSSION TITLE
is commend "yarn create vuetify" create SSR friendly project?

### DISCUSSION THREAD

**Discussion by shahensaifullah** [vuetifyjs/vuetify#16329] [Q&A]
When i write "yarn create vuetify" it create a project. Is this project support SSR? 
if not, what should i do for support SSR?

**Comment by ElijahKotyluk** [comment]
By default the current presets do not enable SSR, there are instructions the V3 docs to enable SSR though:
https://next.vuetifyjs.com/en/getting-started/installation/#ssr

**Reply by shahensaifullah** [reply]
if i use following code

```
const vuetify = createVuetify({
  ssr: true,
})
```
does my project enable as SSR?

**Reply by ElijahKotyluk** [reply]
Yes

**Reply by KaelWD** [reply]
No, that just tells vuetify that the page was server rendered

**Reply by shahensaifullah** [reply]
> No, that just tells vuetify that the page was server rendered

if not. How can i enable SSR in that project?
